lib/web_ui/lib/src/engine/surface/recording_canvas.dart

```diff
@@ -115,7 +120,7 @@ class RecordingCanvas {
   void endRecording() {
     _pictureBounds = _paintBounds.computeBounds();
     if (assertionsEnabled) {
-      _debugRecordingEnded = true;
+      _recordingEnded = true;
     }
```

Comment:
The check for `assertionsEnabled` should be removed, right?

Reply:
Thanks for spotting that!. Done.